packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -60,11 +65,58 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
     };
   }, [cameraSwitcherCallback, cameraSwitcherCameras]);
 
+  const [augmentedParticipants, setAugmentedParticipants] = useState<VideoGalleryRemoteParticipant[] | undefined>(
+    undefined
+  );
+
+  // Async function to retrieve custom data model information to replace display name in remote tiles with camera on.
+  const fetchAvatarPersonaDataAsync = memoizeFunction(
```

Comment:
Let's move this function out of the `MediaGallery` component and avoid creating a new memoized lambda on every re-render